pom.xml

```diff
@@ -108,6 +108,11 @@
       <artifactId>commons-compress</artifactId>
       <version>1.18</version>
     </dependency>
+    <dependency>
```

Comment:
Personally, I don't think this dependency carries its weight. I prefer not to use autovalue.